Makefile

```diff
@@ -704,7 +704,7 @@ define CONFIGURE_LINKER
 
   $(eval $(call CONFIGURE_LINKER_WHOLE,$(1)))
   $(foreach lk,$($(1).links),$(eval $(call CONFIGURE_LIBS,$(lk))))
-  linkcmd += $(libs) $($(1).linkoptions)
+  linkcmd += $(libs) $($(1).linkoptions) -L /usr/local/lib
```

Comment:
why was the /usr/local/lib definition moved ?

Reply:
Make sense. I'd missed the comment. Thanks.

Reply:
I know I had to put it at the beginning for some reason previously. Hopefully it was LLVM related otherwise, this will break something. ¯\_(ツ)_/¯. One way to find out...